pennylane/drawer/draw.py

```diff
@@ -51,8 +74,13 @@ def draw(
             ``None`` will omit parameters from operation labels.
         max_length (int): Maximum string width (columns) when printing the circuit
         show_matrices=False (bool): show matrix valued parameters below all circuit diagrams
+
+    Keyword Args:
+        level (None, str, int, slice): An indication of what transforms to apply before computing the resource information.
+            Check :func:`~.workflow.get_transform_program` for more information on the allowed values and usage details of
+            this argument.
         expansion_strategy (str): The strategy to use when circuit expansions or decompositions
-            are required. Note that this is ignored if the input is not a QNode.
+            are required.

```

Comment:
Similar to [specs](https://xanaduai-pennylane--5855.com.readthedocs.build/en/5855/code/api/pennylane.specs.html), please could we add:
- The note box for `level` vs `expansion_strategy`
- A usage details section to show some examples of setting the `level`.

Reply:
Done! Let me know what you think